stardog/admin.py

```diff
@@ -649,68 +647,91 @@ def validate(self):
         """
         self.client.get('/admin/users/valid')
 
-    #TODO
```

Comment:
Dont we still want to keep the function doc for these?

Reply:
The docs were not removed btw, they were just modified a bit.